database-seeds/scripts/rerunners/add-table-and-folders.js

```diff
@@ -1,13 +1,12 @@
 const { MutateCollection, CreateFolderID } = require("../util");
 
 // Change these for whatever table you are adding.
-const GAME = "pms";
-const PLAYTYPES = ["Controller", "Keyboard"];
-const PREFIX = "PLv";
-const TITLE = "Normal PMS Database";
-const SHORTTITLE = "normalpmsdb"; // this is used in the tableID
-const DESCRIPTION =
-	"The Normal PMS Database. This contains almost every PMS file ever created, lv1-45.";
+const GAME = "ddr";
+const PLAYTYPES = ["SP", "DP"];
+const PREFIX = "diff";
+const TITLE = "DDR A3 Final";
+const SHORTTITLE = "normalpddra3final"; // this is used in the tableID
```

Comment:
likewise for the ID, I think you might've gotten a bit confused because previously this script was used for weird BMS shit. Not really your fault, but this can just be `"a3"`.